src/extra/AutoUi/Lenses/types/entity.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { LensTemplate } from '..';
+import { faCube } from '@fortawesome/free-solid-svg-icons/faCube';
+import { Flex } from '../../../../components/Flex';
+import { Card } from '../../../../components/Card';
+import { Txt } from '../../../../components/Txt';
+import { Heading } from '../../../../components/Heading';
+import { Divider } from '../../../../components/Divider';
+import { AutoUIContext, AutoUIModel } from '../../schemaOps';
+import { Update } from '../../Collection/Actions/Update';
+import { ActionData } from '../../Collection';
+import styled from 'styled-components';
+
+interface EntityProps<T> {
+	entity: T;
+	properties: Array<any & { key: string }>;
+	autouiContext: AutoUIContext<T>;
+	model: AutoUIModel<T>;
+	hasUpdateActions: boolean;
+}
+
+const Label = styled(Txt)`
+	color: #252629;
+	font-size: 11px;
+	text-transform: uppercase;
+	margin-bottom: 6px;
+`;
+
+export const entity: LensTemplate = {
+	slug: 'entity',
+	name: 'Default entity lens',
+	data: {
+		label: 'Entity',
+		format: 'summary',
+		renderer: (props: EntityProps<any>) => {
+			const [actionData, setActionData] = React.useState<
+				ActionData<any> | undefined
+			>();
+			const onActionTriggered = React.useCallback(
+				(actionData: ActionData<any>) => {
+					setActionData(actionData);
+					if (actionData.action.actionFn) {
+						actionData.action.actionFn({
+							affectedEntries: actionData.affectedEntries || [],
+						});
+					}
+				},
+				[],
+			);
+
+			const onActionDone = React.useCallback(() => {
+				setActionData(undefined);
+			}, []);
+
+			return (
+				<Card>
+					{props.entity && (
+						<>
+							<Flex flexDirection="row" justifyContent="space-between">
+								<Flex flexDirection="column">
+									<Heading.h2>
+										{props.properties[0].render(
+											props.entity[props.properties[0].key],
+											props.entity,
+										)}
```

Comment:
what would happen if we pass properties as an empty array ?

Reply:
It would error. I've changed this conditional to the following:
```
{props.properties.length > 0 &&
  props.properties[0].render(
  props.entity[props.properties[0].key],
  props.entity
)}
```